scripts/dev-setup.sh

```diff
@@ -4,15 +4,17 @@ set -euo pipefail
 # This script provisions a working Wormhole dev environment on a minimally provisioned VM.
 # It expects to run as a user without root permissions.
 #
+# It is expected that curl is already installed.
```

Comment:
the purpose of this script is to set up the environment, so we might as well just install curl instead of expecting it to be available. see below

Reply:
Fixed